docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-generic-oidc.md

```diff
@@ -0,0 +1,108 @@
+---
+
+<head> 
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-generic-oidc"/>
+</head>
+
+If your organization uses an OIDC provider for user authentication, you can configure Rancher to allow your users to log in using their IdP credentials. Rancher supports integration using the OpenID Connect (OIDC) protocol and the SAML protocol. Both implementations are functionally equivalent when used with Rancher. This page describes the process to configure Rancher to work using the OIDC protocol.
+
+## Prerequisites
+
+- On Rancher, Generic OIDC is disabled.
+- In your authentication provider, create a new client with the settings below (consult the documentation for your specific provider).
+
+     Setting | Value
+       ------------|------------
+     `Client ID` | &lt;CLIENT_ID> (e.g. `rancher`)
+     `Name` | &lt;CLIENT_NAME> (e.g. `rancher`)
+     `Client Protocol` | `openid-connect`
+     `Access Type` | `confidential`
+     `Valid Redirect URI` | `https://yourRancherHostURL/verify-auth`
+
+- In the new OIDC client, create mappers to expose the users fields (consult the documentation for your specific provider).
+  - Create a new "Groups Mapper" with the settings below.
+
+    Setting | Value
+        ------------|------------
+    `Name` | `Groups Mapper`
+    `Mapper Type` | `Group Membership`
+    `Token Claim Name` | `groups`
+    `Add to ID token` | `OFF`
+    `Add to access token` | `OFF`
+    `Add to user info` | `ON`
+
+  - Create a new "Client Audience" with the settings below.
+
+    Setting | Value
+        ------------|------------
+    `Name` | `Client Audience`
+    `Mapper Type` | `Audience`
+    `Included Client Audience` | &lt;CLIENT_NAME>
+    `Add to access token` | `ON`
+
+  - Create a new "Groups Path" with the settings below.
+
+    Setting | Value
+        ------------|------------
+    `Name` | `Group Path`
+    `Mapper Type` | `Group Membership`
+    `Token Claim Name` | `full_group_path`
+    `Full group path` | `ON`
+    `Add to user info` | `ON`
+
+ - Important:  Rancher will use the value received in the "sub" claim to form the PrincipalID which is the unique identifier in Rancher.  It is important to make this a value that will be unique and immutable.
+
+## Configuring Generic OIDC in Rancher
+
+1. In the Rancher UI, click **☰ > Users & Authentication**.
+1. In the left navigation bar, click **Auth Provider**.
+1. Select **Generic OIDC**.
+1. Complete the **Configure an OIDC account** form. For help with filling the form, see the [configuration reference](#configuration-reference).
+1. After you complete the **Configure an OIDC account** form, click **Enable**.
+
+   Rancher redirects you to the IdP login page. Enter credentials that authenticate with you IdP to validate your Rancher Keycloak configuration.
```

Comment:
```suggestion
   Rancher will redirect you to the IdP login page. Enter your IdP credentials to validate your Rancher Keycloak configuration.
```

I had some trouble interpreting "Enter credentials that authenticate with you IdP". This is just asking you to login, no?

Reply:
> I had some trouble interpreting "Enter credentials that authenticate with you IdP". This is just asking you to login, no?

Yes, just asking you to login